src/models/Rates.js

```diff
@@ -34,6 +34,6 @@ export type Rates = {
   ETH?: number,
 }
 
-export type RatesBySymbol = { [symbol: string]: Rates };
+export type RatesByAssetAddress = { [assetAddress: string]: Rates };
```

Comment:
```suggestion
export type RatesByAddress = { [assetAddress: string]: Rates };
```

Simplier name is descriptive enough, since these are rates for assets.

Reply:
Yes, but the reason I tried to be explicit here (and it other places) is that when it comes to address it can be either Wallet address, Asset address and etc. address. IMO it's better to have explicit which is easier to newcomers to read.